### PR TITLE
Enable VPN variable pricing for wave 1 countries (Fixes #10199)

### LINKED ISSUE #10199: VPN: Variable pricing for wave 1 countries
## Description

Now that variable pricing is available in Germany / France, the VPN team would like to roll out variable pricing to existing wave 1 countries. More details forthcoming...

---

#### Success Criteria
- [x] Wave 1 countries are shown variable pricing options.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Enables variable pricing for VPN in wave 1 countries.
- Feature toggled via `SWITCH_VPN_VARIABLE_PRICING_WAVE_1=on`

## Issue / Bugzilla link
#10199

## Testing
Variable pricing should be shown in US$ for wave 1 countries (clicking "Get Mozilla VPN" should scroll to pricing section):

- [x] http://localhost:8000/en-US/products/vpn/?geo=us
- [x] http://localhost:8000/en-US/products/vpn/?geo=ca
- [x] http://localhost:8000/en-US/products/vpn/?geo=my
- [x] http://localhost:8000/en-US/products/vpn/?geo=nz
- [x] http://localhost:8000/en-US/products/vpn/?geo=sg
- [x] http://localhost:8000/en-US/products/vpn/?geo=gb

Variable pricing should be shown in Euros for wave 2 countries (clicking "Get Mozilla VPN" should scroll to pricing section):

- [x] http://localhost:8000/en-US/products/vpn/?geo=de
- [x] http://localhost:8000/en-US/products/vpn/?geo=fr

Waitlist should be displayed for countries where VPN is not yet available:

- [x] http://localhost:8000/en-US/products/vpn/?geo=es

### CODE REVIEW

**Comment by alexgibson** [comment]
Ok this is ready for code review. Once this merges it will go into QA on www-dev before launching on the 15th June.

File: bedrock/products/templates/products/vpn/includes/pricing-variable.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {% from "products/vpn/includes/macros.html" import vpn_content_block with context %}
 
-<div class="vpn-variable-pricing-block hide-from-legacy-ie">
+<div class="vpn-variable-pricing-block {% if not switch('vpn-variable-pricing-wave-1') %}hide-from-legacy-ie{% endif %}">
```
Comment (craigcook) [inline bedrock/products/templates/products/vpn/includes/pricing-variable.html:7]: Is this correct? This should be **visible** in IE when the switch is **ON**?

**Review by craigcook — APPROVED** [review]
Everything looks good and things render as they should. One question about the switch logic, just to make sure it wasn't an artifact left over from local testing. If not, r+ from me. 🐾 

Maybe @pmac can give a quick once-over of the Python bits, for anything glaring I wouldn't notice since it's outside my domain.

File: bedrock/products/templatetags/misc.py [hunk 2]
```diff
@@ -76,16 +80,14 @@ def vpn_subscribe_link(ctx, entrypoint, link_text, plan=None, class_name=None, l
...
+        pricing_params = settings.VPN_VARIABLE_PRICING.get(lang, settings.VPN_VARIABLE_PRICING['us'])
+        plan_default = pricing_params[plan]['id']
+        plan_attributes = {}
 
         # HTML data-attributes are used by client side JS to set the correct plan ID based on geo.
-        plan_attributes = {
-            'data-plan-de': settings.VPN_VARIABLE_PRICING['de'][plan],
-            'data-plan-fr': settings.VPN_VARIABLE_PRICING['fr'][plan]
-        }
+        for country in settings.VPN_VARIABLE_PRICING.items():
```
Comment (pmac) [inline bedrock/products/templatetags/misc.py:88]: I usually set two variables when using `.items()` just for clarity and readability. e.g. `for country, attrs in settings.VPN_VARIABLE_PRICING.items():`. Not required, but a bit nicer than using just the tuple like this IMO.

**Review by pmac — APPROVED** [review]
Looks good to me for the 🐍 bits. Nice work!

File: bedrock/products/templatetags/misc.py (hunk 2 above)
Comment (alexgibson) [reply bedrock/products/templatetags/misc.py:88]: Nice suggestion! I'll update

**Comment by alexgibson** [comment]
@pmac pushed a commit with the suggested change

**Pull request merged**